test/feature/support/helpers.coffee

```diff
@@ -14,18 +14,19 @@ class Helpers
 
 
   # Creates a new exprestive app in @appPath
-  createExprestiveApp: (optionsStr, done) ->
+  createExprestiveApp: ({library, exprestiveOptions}, done) ->
     serverContents = """
       # Initialize exprestive application
-      express = require 'express'
+      #{library} = require '#{library}'
```

Comment:
the whole `serverContents` is a string that generates a sample expressive application. `library` is a string of either `express` or `connect` based on which http library we want to test expressive with